### DISCUSSION TITLE
How to hide the progress bar?

### DISCUSSION THREAD

**Discussion by markusgermar** [ctreffe/alfred#229] [Alfred Q&A]
Hi there,

I would like to hide/remove the progress bar at the top of each page. How can I do that?

Thank you!
Markus

**Comment by ctreffe** [comment]
Hey @markusgermar,

the progress bar for an alfred3 experiment can be configured either via the config.conf or in a setup hook. You will find a very good description for both methods [here in the documentation](https://alfredo3.psych.bio.uni-goettingen.de/docs/generated/alfred3.experiment.ExperimentSession.progress_bar.html#alfred3.experiment.ExperimentSession.progress_bar).

**Reply by markusgermar** [reply]
Thank you!
